pom.xml

```diff
@@ -23,13 +23,13 @@
         <air.check.skip-spotbugs>true</air.check.skip-spotbugs>
         <air.check.skip-pmd>true</air.check.skip-pmd>
         <air.check.skip-license>true</air.check.skip-license>
-        <air.java.version>1.8.0-40</air.java.version>
+        <air.java.version>21</air.java.version>
 
         <dep.airlift.version>0.216-SNAPSHOT</dep.airlift.version>
         <dep.packaging.version>${dep.airlift.version}</dep.packaging.version>
         <dep.jetty.version>9.4.55.v20240627</dep.jetty.version>
```

Comment:
Can you experiment what all breaks if we upgrade to Jetty 12. [All previous version EOL by 1 Jan 2025](https://endoflife.date/eclipse-jetty)